modules/javafx.graphics/src/main/native-glass/mac/GlassView3D.m

```diff
@@ -777,6 +803,9 @@ - (void)doCommandBySelector:(SEL)aSelector
     // According to Apple an NSResponder will send this up the responder chain
     // but a text input client should not. So we ignore this which avoids an
     // annoying beep.
+    if (keymanActive) {
```

Comment:
I noticed the latest code does not beep.  Were the beeps produced earlier expected for unmapped keys?

Reply:
I'm not sure where the beeps are coming from. I still hear a beep when I press N on the Himyarit Musnad layout but I don't hear a beep when I press X on the ISIS-Devanagari layout. Both are unused keys but one produces a beep and the other doesn't.